src/hybrid/parse/hybrid_optional_args.c

```diff
@@ -110,6 +110,13 @@ int HybridParseOptionalArgs(HybridParseContext *ctx, ArgsCursor *ac, bool intern
                             ctx->reqFlags, sizeof(*ctx->reqFlags), QEXEC_F_SEND_SCORES,
                             ARG_OPT_CALLBACK, handleWithScores, ctx,
                             ARG_OPT_OPTIONAL, ARG_OPT_END);
+
+        // _NUM_SSTRING flag - sets QEXEC_F_TYPED
+        ArgParser_AddBitflagV(parser, "_NUM_SSTRING",
```

Comment:
I guess it is used elsewhere in code, but `_NUM_SSTRING` seems like a very weird name. 

why not `_NO_STRINGIFY`?

Reply:
@nafraf I saw you merged with the existing name. Could be an issue during upgrade if one shard is upgraded and would still receive the old name, so OK with keeping it.

Reply:
we can change it tomrrow quickly if u guys want